docs/index.asciidoc

```diff
@@ -0,0 +1,209 @@
+:plugin: dissect
+:type: filter
+
+[id="plugins-{type}-{plugin}"]
+
+=== Dissect
+
+include::plugin_information.asciidoc[]
+include::../../../logstash/docs/include/plugin_header.asciidoc[]
```

Comment:
I would expect the files related to plugins to live under logstash-docs/docs/plugins/include. Are you planning some bigger structural changes to where files live?